vm/http.go

```diff
@@ -45,12 +47,22 @@ func initResponseClass(vm *VM, hc *RClass) *RClass {
 	hc.setClassConstant(responseClass)
 	builtinHTTPResponseInstanceMethods := []*BuiltInMethodObject{}
 
-	responseClass.setBuiltInMethods(builtinHTTPResponseInstanceMethods, false)
+	responseClass.setBuiltInMethods(builtinHTTPResponseInstanceMethods, true)
```

Comment:
If the second argument is true then those methods will be set as class methods, why change this?
And if you want it to be class methods, please also change `builtinHTTPResponseInstanceMethods` to `builtinHTTPResponseClassMethods`